### PR TITLE
fix(components/lookup): async show more modal toolbars now correctly stick to the top of the modal content

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/408/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/408/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/408/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/408/playground/)

File: libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html [hunk 1]
```diff
@@ -7,7 +7,6 @@
   <sky-modal-content
     [skyViewkeeper]="['.sky-lookup-show-more-toolbar-wrapper']"
   >
-    <sky-wait [isWaiting]="isSearching"></sky-wait>
```
Comment (Blackbaud-JohnHolland) [inline libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html]: was this intentionally removed?

File: libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html [hunk 2]
```diff
@@ -81,41 +80,41 @@
...
-        </sky-repeater-item>
-      </sky-repeater>
-      <sky-infinite-scroll
-        [enabled]="hasMoreItems"
-        [loading]="isLoadingMore"
-        (scrollEnd)="infiniteScrollEnd()"
-      >
-      </sky-infinite-scroll>
     </div>
+    <div
```
Comment (Blackbaud-JohnHolland) [inline libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html:85]: from what I can tell this code is the same, it's just the indentation that's different?? I might be missing the changed line.

**Review by Blackbaud-JohnHolland — COMMENTED** [review]
couple of questions

File: libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html (hunk 2 above)
Comment (Blackbaud-StephenRicharme) [reply libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html:85]: Previously, the `<div>` for "show-no-more-results" and the accompanying `<sky-repeater>` and `<sky-infinite-scroll>` elements were children of the `<div class="sky-lookup-show-more-toolbar-wrapper">` element.  Now they are siblings of that element and all children of the `<sky-modal-content>` element.  So it's more than just indentation - it's the parent/child relationship of the elements.

File: libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.html]: Nope. Great catch. I removed during testing and accidentally checked it in. Fixed.

**Review by Blackbaud-StephenRicharme — APPROVED** [review]

**Review by Blackbaud-JohnHolland — APPROVED** [review]

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**